pkg/drain/drain.go

```diff
@@ -483,12 +486,15 @@ func deletePods(client typedcorev1.CoreV1Interface, pods []corev1.Pod, options *
 	} else {
 		globalTimeout = options.Timeout
 	}
-	deleteOptions := &metav1.DeleteOptions{}
-	if options.GracePeriodSeconds >= 0 {
-		gracePeriodSeconds := int64(options.GracePeriodSeconds)
-		deleteOptions.GracePeriodSeconds = &gracePeriodSeconds
-	}
 	for _, pod := range pods {
+		deleteOptions := &metav1.DeleteOptions{}
+		if options.GracePeriodSeconds >= 0 {
```

Comment:
can we move the duplicated logic into a function, reuse it in both places and add unit test?

Reply:
done